Readme fixes

supertom opened the pull request:
Updated repo and config to support latest stable (2015.8.8).

FYI @erjohnso 

*Summary of changes:*
 - update version to 2015.8.8
 - update to use JSON key instead of p12; removed openssl command
 - added salt-cloud -u command to update bootstrap script
 - add ssh_username: saltuser to profile
 - add master_type: standard to profile
 - minion install with saltuser@
 - change "provider" to "driver" in /etc/salt/cloud
 - updated console links
 - removed overwrite step of gce.py
 - spell-check

erjohnso commented on README.md:
```diff
@@ -18,48 +18,50 @@ At the end of the demo, you will have used SaltStack to automate:
-ends with `@developer.gserviceaccount.com` since this will be required in the
-Salt configuration files.
+for your Service Account as you will need to upload it to your Salt master later on. Also, make sure 
+to record the *Email address* that ends with `@developer.gserviceaccount.com` since 
```
Is this still true for new Service Accounts? I thought now it was @iam.something.or.other.com.

supertom replied:
For the default account, it remains at developer.gserviceaccount.com.  For new accounts, it's project-id.iam.gserviceaccount.com.  Your asking this question made me realize that creating a new account his not only safer but allows us to have a little more control over the name, and since they had to create a new account previously, it's isn't any more work on their end.  So, I changed it.

erjohnso commented on README.md:
````diff
@@ -75,85 +77,93 @@ salt us-central1-b n1-standard-1 10.240.136.204 123.45.67.89    RUNNING
     ```
 
-1. Install libcloud (v0.14.1 or greater)
+1. Install libcloud (v1.0.0-rc2 or greater)
````
Is there something 1.0.0 specific that's needed for this now, or would an older version of libcloud still work?

supertom replied:
As of this writing, 1.0.0-rc2 is what's pulled in.  However, looking at the salt-cloud docs, the last version they reference is 0.14.0-dev (before they merged it with the salt repo), and their latest docs don't specify a version at all, just a pip install.

I changed it back to 0.14.1 as to not give the impression, as you received, that we've done something different that needs an updated libcloud version.

erjohnso commented on README.md:
````diff
@@ -170,17 +180,19 @@ to customize them with your credentials.
     ```
-    openssl pkcs12 -in /path/to/original/key.p12 -passin pass:notasecret -nodes -nocerts | openssl rsa -out /etc/salt/pkey.pem
-
+    openssl pkcs12 -in /home/USER/project-id-0375632f3fe4.p12 -passin pass:notasecret -nodes -nocerts | openssl rsa -out /etc/salt/pkey.pem
````
TODO(@supertom): make this work with the JSON credentials file so users don't have to type this archaic openssl command.  Also, with a master deployed in GCE, it should be able to use libcloud's "internal" auth using metadata tokens.

supertom replied:
Done!  We're flying with JSON keys now!

supertom commented:
@erjohnso Thanks for the quick review and suggestions.  PTAL.

Changes:
- Users create new service account, makes naming more consistent
- Using the JSON private key now: updated docs and removed openssl command, changed configs
- libcloud version back to 0.14.1

erjohnso commented:
lgtm, ship it!
